infra/token-transfer-client/src/components/pages/LockupConfirm.js

```diff
@@ -45,7 +45,15 @@ const LockupConfirm = props => {
   return (
     <div className="text-center p-5 text-muted">
       <h1>
-        An error occurred confirming your token lockup, has the token expired?
+        An error occurred confirming your token lockup. Please try again or{' '}
```

Comment:
99% of the time this will be an expiry, you sure we don't want to mention something about expiry?

Reply:
How about something like the following:

> An error occurred with your token lockup. Please try again and be sure to confirm the email within 10 minutes of receiving it.
>
> Contact Support

⌛️

Reply:
Yea that sounds good 👍